### PR TITLE
HAWQ-1629. Add ORC format using pluggable storage framework.

### PR DESCRIPTION
**Pull request by oushu1wangziming1**
HAWQ-1629. Add ORC format using pluggable storage framework.

### CODE REVIEW

**Comment by radarwave** [comment]
Please check licenses and make sure you passed RAT checks.

**Comment by oushu1wangziming1** [comment]
Well. I add the licenses in orc.c

File: contrib/orc/README [hunk 1]
```diff
@@ -0,0 +1,84 @@
+Fresh installation from 3.0.0.0 source code
```
Comment (radarwave) [inline contrib/orc/README]: The version is confused.

File: contrib/orc/README [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+2. Configure and initialize cluster
+   $ hawq init cluster -a
+
+
+
+Fresh installation from 3.0.0.0 rpm package
+-------------------------------------------
+1. Setup 3.0.0.0 yum repository
+   $ cd /etc/yum.repos.d
+   $ sudo wget  http://yum.oushu.io/oushurepo/yumrepo/release/oushu-database/centos7/3.0.0.0/oushu-database.repo
```
Comment (radarwave) [inline contrib/orc/README]: Is this repo required? Any more general repo url?

**Comment by huor** [comment]
+1

**Comment by radarwave** [comment]
+1

**Pull request closed without merging**